dev-docs/bidders/appnexus-ast.md

```diff
@@ -19,6 +19,10 @@ adapter include:
 - Will continue to be upgraded as enhancements are made to Prebid.js
   for expanding capabilities
 
+{: .alert.alert-danger :}
+**All AppNexus placements on the page must come from the same publisher**  
```

Comment:
"All AppNexus placements included in a single call to `requestBids` must belong to the same Publisher object"